view/src/main/webapp/WEB-INF/jsp/proposals/proposalDetails_edit.jspx

```diff
@@ -114,12 +114,6 @@
         </div> <!-- /c-Headline -->
 
 
-        <form action="${contest.proposalLogoPath}images/upload" method="post" enctype="multipart/form-data" target="_fileUploadFrame" id="fileUploadForm">
-            <input type="hidden" name="${_csrf.parameterName}" value="${_csrf.token}"/>
-            <input type="file" name="file" id="fileUploadInput" />
-        </form>
-
-
```

Comment:
Is this change related?

Reply:
Indirectly - it was using the method that I changed. This code would have caused an error after changing the method, but since it's unused I removed it.

Reply:
Okay, got it.